src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -15,45 +15,81 @@ import {
   FacetRefinement,
   NumericRefinement,
 } from '../../lib/utils/getRefinements';
-import { Connector, TransformItems } from '../../types';
+import { Connector, TransformItems, CreateURL } from '../../types';
 
-export type ConnectorRefinement = {
+export type CurrentRefinementsConnectorParamsRefinementType =
+  | 'facet'
+  | 'exclude'
+  | 'disjunctive'
+  | 'hierarchical'
+  | 'numeric'
+  | 'query'
+  | 'tag';
+
+export type CurrentRefinementsConnectorParamsRefinement = {
+  /**
+   * The attribute on which the refinement is applied.
+   */
   attribute: string;
-  type: string;
+
+  /**
+   * The type of the refinement.
+   */
+  type: CurrentRefinementsConnectorParamsRefinementType;
```

Comment:
This was a string before, please confirm that is indeed one of CurrentRefinementsConnectorParamsRefinementType strings.

Reply:
[Yes](https://www.algolia.com/doc/api-reference/widgets/current-refinements/js/#connector-param-render-items).